File: extensions/vscode-jetbrains-color-theme/assets/examples/sample.go
// Single-line comment
/* Multi-line
   comment */
   package main

   import (
	   "fmt"
	   "math"
   )

   type Shape interface {
	   Area() float64
   }

   type Circle struct {
	   radius float64
   }

   func (c *Circle) Area() float64 {
	   return math.Pi * c.radius * c.radius
   }

   func main() {
	   ch := make(chan int)
	   go func() {
		   ch <- 42
	   }()

	   defer fmt.Println("Done")

	   if num := <-ch; num > 0 {
		   fmt.Printf("Value: %v", num)
	   }
   }

File: extensions/vscode-jetbrains-color-theme/assets/examples/sample.cpp
// Single-line comment
/* Multi-line
   comment */
   #define MACRO 10

   #include <iostream>
   using namespace std;
   
   template<typename T>
   class MyClass : public ParentClass {
   private:
       T* data;

   public:
       MyClass(T value) noexcept : data(new T(value)) {}

       friend ostream& operator<<(ostream& os, const MyClass& obj);
   };

   auto lambda = [](auto x, auto y) { return x + y; };

   int main() {
       vector<int> numbers = {1, 2, 3};
       for (auto& num : numbers) {
           cout << num << endl;
       }
       return 0;
   }
